### PR TITLE
[ADD] module FDS PostFinance

### PR DESCRIPTION
**Pull request by ndtran**
Depends on #147 
# Modules: 
- l10n_ch_fds_postfinance (allow users to import files from FDS PostFinance and convert imported files to bank statment)
- l10n_ch_fds_upload_dd (allow users to upload direct debit order files to FDS PostFinance)
- l10n_ch_fds_upload_sepa (allow users to upload sepa files to FDS PostFinance)
# Postfinance File Delivery Services

The file delivery services (FDS) is service offered by Postfinance AG Technology unit service.

FDS acts as a gateway between external networks and the Post CH SA. It enables mutual exchange of files between partners and Post CH applications.

### CODE REVIEW

**Comment by ecino** [comment]
Can you rebase on OCA/8.0 ? It should remove Travis errors.

**Comment by yvaucher** [comment]
@ndtran  Warning #147 was superseeded by #162

File: l10n_ch_fds_postfinance/README.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+Postfinance File Delivery Services
+==================================
+The file delivery services (FDS) is service offered by Postfinance AG Technology unit service.
+
+FDS acts as a gateway between external networks and the Post CH SA. It enables mutual exchange of files between partners and Post CH applications.
+
+Benefit
+-------
+This module allow Odoo users to import files on their FDS PostFinance and convert imported files to bank statment.
```
Comment (yvaucher) [inline l10n_ch_fds_postfinance/README.md:12]: s/allow/allow**s**

File: l10n_ch_fds_postfinance/README.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Postfinance File Delivery Services
+==================================
+The file delivery services (FDS) is service offered by Postfinance AG Technology unit service.
```
Comment (yvaucher) [inline l10n_ch_fds_postfinance/README.md:6]: s/is service/is a service

**Comment by yvaucher** [comment]
I did most of my comments on the last commit as it was easier to see what is added in this PR

Here we go for the code review.

**Comment by yvaucher** [comment]
As it is superseeded by #188 I close this

**Pull request closed without merging**